crt_portal/api/views.py

```diff
@@ -366,7 +366,7 @@ class ResponseList(generics.ListAPIView):
 
 
 class ResponseDetail(generics.RetrieveAPIView):
-    queryset = ResponseTemplate.objects.all()
+    queryset = ResponseTemplate.objects.filter(show_in_dropdown=True).all()
```

Comment:
Maybe a nitpick, but I think we want to remove the `.all()` here? I don't think .all() is necessary after calling filter. I don't know if it hurts anything either, but may want to remove it just in case?